accounts/tests.py

```diff
@@ -14,63 +14,150 @@
 from django.test import TestCase, override_settings
 from django.contrib.auth.models import Group
 import logging
+from django.core.management import call_command
+from django.utils.six import StringIO
+from django.contrib.auth.models import User
+
+
+TEST_PIPELINE = (
+    'social.pipeline.social_auth.social_details',
+    'social.pipeline.social_auth.social_uid',
+    'social.pipeline.social_auth.auth_allowed',
+    'social.pipeline.social_auth.social_user',
+    'social.pipeline.user.create_user',
+    'accounts.pipelines.save_profile',
+    'social.pipeline.social_auth.associate_user',
+    'social.pipeline.social_auth.load_extra_data',
+    'social.pipeline.user.user_details',
+)
+
+from .utils import create_user 
+from django.contrib.auth.models import User
+from django.db import connection
+
+class Utilities(LiveServerTestCase, WagtailPageTests):
+    serialized_rollback = True
+
+    def setUp(self):
+        super(WagtailPageTests, self).setUp()
+        super(LiveServerTestCase, self).setUp()
+
+    def test_import_user(self):
+        user_details = {'last_name': 'Hart', 
+                        'username': 'openstax_cms_tester', 
+                        'full_name': None, 
+                        'first_name': 'Richard',
+                        'uid': 16180}
+        # check user doesnt exist
+        if User.objects.filter(username=user_details['username']).exists():
+            User.objects.filter(username=user_details['username'])[0].delete()
+        cursor = connection.cursor()
+        cursor.execute("SELECT username FROM auth_user")
```

Comment:
Instead of raw sql here, can you do Users.objects.all()? I think you will be able to do a assertNotIn on the results the same way you are below.